### ISSUE TITLE
[WFRP4e System] Group Advantage not working on Character Sheet

### ISSUE THREAD

**Issue by moo-man** [moo-man/WFRP4e-FoundryVTT#1945]
There are several issues.

Setup:
1. Group Advantage is enabled
2. There is combat with 2 Actors of "Player Character" type. One is assigned to a user, one is of "Neutral" Disposition
3. Advantage in Combat Tracker is set to 3-2

Issues:
1. Both Actor sheets (again, both are PC type) display `0` as their advantage
2. Trying to edit "Group Advantage" fields on Actor Sheet results in `This actor must be part of a Combat in order to update group advantage.` notice. 
3. Trying to use any action, such as "Batter" results in `Not enough Advantage!` error

<br/>**From**: forien<br/>foundry: 11.315<br/>wfrp4e: 7.1.2<br/>wfrp4e-core: 5.0.2<br/>wfrp4e-up-in-arms: 5.0.0<br/>wfrp4e-wom: 5.0.1<br/>Active Modules: ATL, chat-commander-wfrp4e, forien-actors-summary, forien-ammo-swapper, forien-armoury, forien-easy-polls, foundryvtt-simple-calendar, lib-wrapper, quick-insert, sidebar-macros, socketlib, times-up, token-action-hud-core, token-action-hud-wfrp4e, _chatcommands

**Comment by Forien** [comment]
Issue no. 1 can be fixed with this change:

https://github.com/moo-man/WFRP4e-FoundryVTT/blob/2a5870dfda4c084a37fd03bb575f8c8e7551cd55/modules/model/actor/standard.js#L287-L290
```diff
- let combatant = game.combat?.getCombatantByActor(this);
+ let combatant = game.combat?.getCombatantByActor(this.parent);
```

However, this only allows to set personal character's advantage to a number (which will update "Group Advantage"), but they are still disconnected. Changes to Group Advantage do not affect Character.

So I can set Group Advantage to `6`, but Actor still has `0`, it doesn't see the `6` and I cannot use Group Advantage Actions.
I can however, on Actor set Advantage to `1`, which will now change Group Advantage from `6` to `1`. Then I can use Batter, which will reduce both to `0`, but if I change Advantage in Combat back to let's say `2`, Actor remains at `0`.

Basically, something prevents combat advantage pool from updating the Actor. Haven't tested latest commits, so it may be further fixed as it seems to be related to #1937 

Will update code and test again

**Comment by Forien** [comment]
Okay, The new code is better, but somehow now it seems even weirder.

Ok, so I updated the code and changed line 289 as I showed previously.

And while global updates do reflect on Actor correctly, Actor after a while stops updating itself. Something prevents proper Update. Video will showcase it better that my english:

https://i.gyazo.com/a47e99acdfe64c51af24f317b312b9c3.mp4
